addons/event_sale/wizard/event_edit_registration.py

```diff
@@ -17,10 +18,18 @@ def default_get(self, fields):
             sale_order_id = res.get('sale_order_id', self._context.get('active_id'))
             res['sale_order_id'] = sale_order_id
         sale_order = self.env['sale.order'].browse(res.get('sale_order_id'))
-        registrations = self.env['event.registration'].search([
+        domain = [
             ('sale_order_id', '=', sale_order.id),
             ('event_ticket_id', 'in', sale_order.mapped('order_line.event_ticket_id').ids),
-            ('state', '!=', 'cancel')])
+            ('state', '!=', 'cancel')]
+        registrations = self.env['event.registration'].search(domain)
+
+        event_count = self.env['event.registration'].read_group(
+            domain, ['event_id'], ['event_id'],
+        )
+
+        #only show the event collumn if there are multiple events
```

Comment:
column*

Reply:
wording corrected in the compute method!